### PR TITLE
chore: hardcode optimism testnet for retrieving communities portal

### PR DESCRIPTION
**Pull request by richard-ramos**
It does not require mobile testing as it is only used on Desktop

### CODE REVIEW

File: protocol/messenger_communities.go [hunk 1]
```diff
@@ -208,7 +210,7 @@ func (m *Messenger) JoinedCommunities() ([]*communities.Community, error) {
 }
 
 func (m *Messenger) CuratedCommunities() (*communities.KnownCommunitiesResponse, error) {
-	testNetworksEnabled, err := m.settings.TestNetworksEnabled()
+	/*testNetworksEnabled, err := m.settings.TestNetworksEnabled()
```
Comment (0x-r4bbit) [inline protocol/messenger_communities.go]: Shall we remove this then?

File: protocol/messenger_communities.go [hunk 2]
```diff
@@ -222,6 +224,34 @@ func (m *Messenger) CuratedCommunities() (*communities.KnownCommunitiesResponse,
 	if err != nil {
 		return nil, err
 	}
+	*/
+
+	// --- Delete this code once we have the curated communities contract in mainnet
+	// Since we are bypassing tesnet being enabled or not, we gotta obtain the networks from the node config
+	chainID := uint64(69)
```
Comment (0x-r4bbit) [inline protocol/messenger_communities.go]: Oh , this optimisms chain ID? Maybe add a comment for that?

**Review by 0x-r4bbit — APPROVED** [review]

**Comment by qoqobolo** [comment]
> It does not require mobile testing as it is only used on Desktop

Got it. Will mark it as `tested` for better visibility

File: protocol/messenger_communities.go (hunk 1 above)
Comment (richard-ramos) [reply protocol/messenger_communities.go]: yeah, will remove it and add a comment containing the commit that contained the code before the change so it's easier to revert without having to browse the commit history

**Pull request merged**